src/main/java/Deadline.java

```diff
@@ -0,0 +1,25 @@
+/**
+ * Represents a deadline task in the list.
+ */
+public class Deadline extends Task {
+    private String time;
+
+    /**
+     * Creates a deadline task.
```

Comment:
Is it better if we can have an empty line to separate the description and the @param?